apps/storybook/src/gp2-components/IconWithLabel.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import projectIcon from '@asap-hub/gp2-components/src/icons/project-icon';
+import IconWithLabel from '@asap-hub/gp2-components/src/molecules/IconWithLabel';
+import { text } from '@storybook/addon-knobs';
+
+export default {
+  title: 'GP2 / Molecules / Icon With Label',
+  component: IconWithLabel,
+};
+export const Logo = () => (
+  <IconWithLabel icon={projectIcon}>
+    {text(
+      'description',
```

Comment:
I would rename this to `label` and the text could just be "label", doesn't make too much sense being a big text